blacksheep/testing/helpers.py

```diff
@@ -9,6 +9,8 @@ def get_example_scope(
     scheme: str = "http",
     server: Optional[List] = None,
 ):
+    """Returns mocked ASGI scope"""
+
     if "?" in path:
         raise ValueError("The path in ASGI messages does not contain query string")
```

Comment:
Since you are moving this piece of code from `tests` under the main namespace, it is now analyzed for code coverage. The only missing line for this file is the `"?" in path` check (bringing down the coverage for this file to 89%). So we should either add a test to assert that `get_example_ scope` throws if the path contains a query string, or eventually replacing the line with an assertion:

```python
assert "?" not in path, "The path in ASGI messages does not contain query string"
```

Reply:
Nice catch! will do